inc/namespace.php

```diff
@@ -89,21 +109,24 @@ function render_settings_description(): void {
 
 /**
  * Render the settings field UI.
+ *
+ * @param array $args Field callback args.
  */
-function render_field_ui(): void {
+function render_field_ui( array $args ): void {
```

Comment:
I would prefer to have dedicated functions for rendering each field, to be honest. 🙂